util/doxygen/Doxyfile

```diff
@@ -0,0 +1,203 @@
+# Doxyfile 1.8.17
```

Comment:
Did you consider using MAX_INITIALIZER_LINES = 0 to avoid the use of `hideinitializer` in the docstrings?  What is the general policy wrt the use of {hide,show}initializer?

Reply:
In general, and especially in the DIFs, I would like all the initializers to be explicitly shown, so they can be correlated against hardware constants.

I am explicitly using `@hideinitializer` in our freestanding C headers, because many of the definitions use internal compiler constructs that I do not want to be shown in the documentation (but I do want the presence of these definitions to be documented).

I'm not sure there is a way of doing hide/show initializer on a per-file basis, which could solve this issue in a nicer way.